hwilib/devices/coldcard.py

```diff
@@ -242,6 +242,10 @@ def prompt_pin(self):
     def send_pin(self, pin):
         raise UnavailableActionError('The Coldcard does not need a PIN sent from the host')
 
+    # Toggle passphrase
+    def toggle_passphrase(self):
+        raise NotImplementedError('The Coldcard does not implement this method')
```

Comment:
To be consistent with other commands, this should be `UnavailableActionError` unless it is possible to implement this feature for the other devices.

Reply:
Thanks @achow101, I modified the error for `ColdCard`, `Ledger` and `Digital Bitbox`, and squashed the commits.